docs/okeps/okep-5088-evpn.md

```diff
@@ -0,0 +1,893 @@
+  advertise-all-vni
+ exit-address-family
+exit
+```
+
+Notice the vrf stanzas are no longer needed.
+
+Architecturally, the traffic pattern and topology will look the same as the diagram in the
+[MAC-VRF + IP-VRF Combination with Layer 2 UDN](#l2evpn-anchor) section.
+
+### Feature Compatibility
+
+#### Multiple External Gateways (MEG)
+
+Not supported.
+
+#### Egress IP
+
+Not supported.
+
+#### Services
+
+Full support for cluster IP access from pods. Limited support for external service access (node port, external IP,
+load balancer IP).
+MetalLB does not currently have a way to specify per VRF advertisements to the same BGP peer. Therefore,
+for users to advertise an external IP or LoadBalancer IP, they must configure FRR-K8S to manually advertise it. A user
+may also configure FRR-K8S to advertise cluster IP as well externally into the EVPN fabric.
+
+MetalLB may be extended in the future to support per-VRF advertisement.
+
+#### Egress Service
+
+Not supported.
+
+#### Egress Firewall
+
+Full support.
+
+#### Egress QoS
+
+Full support.
+
+#### Network Policy/ANP
+
+Full support.
+
+#### IPSec
+
+Not supported.
+
+#### Multicast
+
+IP-VRF will not be supported for Multicast. Although OVN can handle router forwarding of multicast traffic, the kernel
+routing will not be configured correctly. Multicast within IP-VRF is a less common use case than MAC-VRF. Multicast
+within a MAC-VRF will be supported. The OVN logical switch will handle IGMP snooping and forwarding unknown multicast
+with the logical_switch configuration set to `other_config:mcast_flood_unregistered=true` and `other_config:mcast_snoop=true`.
+The linux bridge will only have the VXLAN VTEP and OVN internal port connected to it, so it is not necessary to configure
+IGMP snooping there. Type 3 EVPN routes will be sent to announce VPN/VTEP membership and trigger ingress replication to
+flood multicast packets to the right VTEPs across the fabric. The linux bridge will simply "flood" the packet to the OVS
+internal port to the OVN logical switch. The OVN logical switch will then rely on IGMP snooping to limit which pods to
+send the packet to.
+
+### Testing Details
+
+The EVPN feature will require E2E tests to be written which will simulate a spine and leaf topology that the KIND Kubernetes
+nodes are attached to. From there tests will be added that will create UDN+BGP+EVPN and test the following:
+
+1. UDN pods are able to talk to external applications on the same VPN.
+2. UDN pods are unable to talk to external applications on a different VPN.
+3. UDN pods are able to talk to other UDN pods on the same network without a Geneve overlay via EVPN.
+4. The above tests will apply for both IP-VRF and MAC-VRF EVPN types.
+5. For IP+MAC-VRF, a test will be added to ensure VM migration between two Kubernetes nodes with EVPN. This includes
+ensuring TCP connections are not broken as well as minimal packet loss during migration.
+6. Testing with EgressIP.
```

Comment:
I'm confused. Didn't you say this was not supported ? 
https://github.com/ovn-kubernetes/ovn-kubernetes/pull/5089/files#diff-1e9b068f81697f43f7b344012d1ce60e07befe8e3d19c471c9e24b029e628763R771